pkg/cmd/pr/shared/comments.go

```diff
@@ -81,20 +103,76 @@ func formatComment(io *iostreams.IOStreams, comment api.Comment, newest bool) (s
 	fmt.Fprintln(&b)
 
 	// Reactions
-	if reactions := ReactionGroupList(comment.ReactionGroups); reactions != "" {
+	if reactions := ReactionGroupList(comment.Reactions()); reactions != "" {
 		fmt.Fprint(&b, reactions)
 		fmt.Fprintln(&b)
 	}
 
 	// Body
-	if comment.Body != "" {
-		style := markdown.GetStyle(io.TerminalTheme())
-		md, err := markdown.Render(comment.Body, style, "")
-		if err != nil {
-			return "", err
-		}
-		fmt.Fprint(&b, md)
+	content := comment.Content()
+	if content == "" {
+		content = "_No body provided_"
```

Comment:
That is odd. `No body provided` should be in italics as the screenshot above shows. Perhaps the font you are using does not have an italics version? Or your terminal does not support italics?

Reply:
It's probably my terminal. Since italics are not widely supported in terminals, I wonder if we could have another visual indicator? Perhaps something like `(no body provided)` printed in gray.

Reply:
I am not opposed to changing this. We would probably also want to change the issue/PR `No description provided` string style to match whatever we do here. I know there has been some issues with gray not being very visible in the past that we should take into account. @ampinsk do you have an opinion?

Reply:
We have fixed issues with gray. It's much more likely that italics are not available in a terminal. Here's the result of my quick testing whether italics are available with default settings:

- ✅ macOS Terminal.app
- ❌ macOS iTerm - toggle to enable available in settings
- ❌ Windows cmd.exe
- ❌ Windows Powershell
- ❌ Windows Terminal
- ❌ Windows Conemu
- ✅ Windows Git Bash (mingw64)
- for Linux, [see here](https://superuser.com/a/958804) (from 2015)

Reply:
Thanks for investigating. Due to the lack of windows support I removed the italics and changed the color to gray.